build/configs/rtl8721csm/board_metadata.txt

```diff
@@ -26,5 +26,3 @@ BL2="km4_boot_all"
 KERNEL="km0_km4_image2"
 OTA="km0_km4_image2"
 BOOTPARAM="bootparam"
-
-FLASH_START_ADDR=0x08000000
```

Comment:
FLASH_START_ADDR is used in common_download.sh. Please modify common_download.sh accordingly.
https://github.com/Samsung/TizenRT/blob/77f1253131f4e8fafd827a0bf4accfbf219c6139/build/configs/common_download.sh#L404